### PR TITLE
Keycode -> Util::Keycode

### PR DESCRIPTION
**Pull request by ntut-rick**
fix typo _from https://github.com/NOOBDY/practical-tools-for-simple-design/commit/3665f83b1a855987906858522079a9c6cf6c0fd9 in #7_

### CODE REVIEW

**Comment by NOOBDY** [comment]
I think this is fixed in #9 ?

**Comment by ntut-rick** [comment]
But we block it for other reasons. And we can split tasks as small as possible.

**Review by Error0229 — APPROVED** [review]

File: src/App.cpp [hunk 1]
```diff
@@ -6,6 +6,8 @@
 #include "Util/Keycode.hpp"
 #include "Util/Logger.hpp"
 
+using Util::Keycode;
+
```
[the remark this answers is not in the document]
Comment (x06lan) [reply src/App.cpp:10]: You need to stop wick brainwash another people approval PR

[the remark this answers is not in the document]
Comment (Error0229) [reply src/App.cpp:10]: This is coding style problem.

[the remark this answers is not in the document]
Comment (NOOBDY) [reply src/App.cpp:10]: > You need to stop wick brainwash another people approval PR

what?

**Review by Dragonfly911117 — COMMENTED** [review]
Don't think this PR is necessary but merge it if needed.

**Comment by NOOBDY** [comment]
actually for my [`workflow`](https://github.com/NOOBDY/practical-tools-for-simple-design/tree/workflow) branch to succeed this needs to be merged

**Comment by Dragonfly911117** [comment]
Then, do it

**Review by NOOBDY — CHANGES_REQUESTED** [review]
Util::Keycode::ESCAPE

**Review by NOOBDY — APPROVED** [review]
LGTM

**Pull request merged**